backend/app/libs/storage_helpers.py

```diff
@@ -19,6 +19,17 @@ def retrieve_treasuries_metadata() -> list[tuple[str, int]]:
     return [tuple(json.loads(t).values()) for t in db.lrange("treasuries", 0, -1)]
 
 
+def store_token_whitelist(address: str):
```

Comment:
Can't we rather use a Redis set?

👓 https://redis.io/docs/manual/data-types/

Reply:
Thanks, this will give us a big performance improvements 👍.